finetune/base.py

```diff
@@ -358,21 +365,18 @@ def _cached_inference(self, Xs, mode=None):
 
     def _inference(self, Xs, mode=None):
         Xs = self.input_pipeline._format_for_inference(Xs)
-        if self._cached_predict:
-            return self._cached_inference(Xs=Xs, mode=mode)
-        else:
-            estimator, hooks = self.get_estimator()
-            input_fn = self.input_pipeline.get_predict_input_fn(Xs)
-            length = len(Xs) if not callable(Xs) else None
-
-            predictions = tqdm.tqdm(
-                estimator.predict(
-                    input_fn=input_fn, predict_keys=mode, hooks=hooks
-                ),
-                total=length,
-                desc="Inference"
-            )
-            return [pred[mode] if mode else pred for pred in predictions]
+        length = len(Xs) if not callable(Xs) else None
```

Comment:
Did we entirely remove the `_cached_predict` functionality?

Reply:
Deployment Model is by default cached.